Documentation/devel/stage1-implementors-guide.md

```diff
@@ -12,9 +12,9 @@ rkt's execution of pods is divided roughly into three separate stages:
 
 This separation of concerns is reflected in the file-system and layout of the composite image prepared by stage 0:
 
-0. Stage 0: `rkt` executable, and the Pod Manifest created at "/var/lib/rkt/pods/$uuid/pod".
-1. Stage 1: "stage1.aci", made available at "/var/lib/rkt/pods/$uuid/stage1" by `rkt run`.
-2. Stage 2: "$app.aci", made available at "/var/lib/rkt/pods/$uuid/stage1/rootfs/opt/stage2/$appname" by `rkt run`, where $appname is the name of the app in the pod manifest.
+0. Stage 0: `rkt` executable, and the Pod Manifest created at `/var/lib/rkt/pods/$uuid/pod`.
```

Comment:
Isn't this path actually `/var/lib/rkt/pods/run/$uuid/pod`?

And same for the next two lines (missing `run`).